andela/metis-ah-frontend
# #161963307 Fix bug on user profile article section

#### What does this PR do?
Show the right message on the article section on the user profile

#### Description of Task to be completed?
- Display the right message by comparing the id of the logged in user and id of the user trying to access that page

#### How should this be manually tested?
- clone this repo
- run `npm install` to install dependencies
- run `npm run dev` to start the development server
- login into the app
- visit your profile and visit another user's profile to see the difference

#### Any background context you want to provide?
N/A

#### What are the relevant pivotal tracker stories?
[161963307](https://www.pivotaltracker.com/story/show/161963307)

#### Screenshots (if appropriate)
This is a screenshot of the message displayed when a user visits the page of another user that has not created an article
<img width="1189" alt="screen shot 2018-11-15 at 1 13 25 pm" src="https://user-images.githubusercontent.com/26192691/48552399-cb86f700-e8d8-11e8-9038-0ed6f04ef189.png">


This is the screenshot of the fix
<img width="1109" alt="screen shot 2018-11-15 at 1 10 58 pm" src="https://user-images.githubusercontent.com/26192691/48552409-d3469b80-e8d8-11e8-9b8d-89d1e30e8131.png">


#### Questions:
N/A
